src/ConfiguresApplication.php

```diff
@@ -151,16 +148,7 @@ protected function writeEnvironmentVariable($key, $value)
      */
     protected function reloadEnvironment()
     {
-        $adapters = [
-            new EnvConstAdapter(),
-            new PutenvAdapter(),
-            new ServerConstAdapter(),
-        ];
-
-        $repository = RepositoryBuilder::create()
-            ->withReaders($adapters)
-            ->withWriters($adapters)
-            ->make();
+        $repository = Env::getRepository();
```

Comment:
This change addressed the following error that I was seeing, and should make this package more resilient to any future changes in how Laravel (or DotEnv) parse the environment:

<img width="984" alt="Screen Shot 2021-07-14 at 3 41 47 PM" src="https://user-images.githubusercontent.com/583202/125684089-77f8a451-dcd6-4376-a73c-c5081395d60f.png">